Helmfile SIGSEGV on many commands

I'm getting this error on almost every helmfile invocation.

Just migrated from 0.21.0 to 0.33.2.

```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
        panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0x74e2d7]

goroutine 1 [running]:
github.com/roboll/helmfile/vendor/github.com/urfave/cli.HandleAction.func1(0xc420148f48)
        /go/src/github.com/roboll/helmfile/vendor/github.com/urfave/cli/app.go:474 +0x279
panic(0x7d0de0, 0xa78ae0)
        /usr/local/go/src/runtime/panic.go:502 +0x229
github.com/roboll/helmfile/args.GetArgs(0x0, 0x0, 0x0, 0x0, 0x0, 0xc420148718)
        /go/src/github.com/roboll/helmfile/args/args.go:65 +0x87
main.executeDiffCommand(0xc4200c8c80, 0x0, 0x0, 0x0, 0xc420150000, 0x0, 0x0, 0x1da0, 0x1fa0, 0xc42016c000, ...)
        /go/src/github.com/roboll/helmfile/main.go:517 +0x7a
main.main.func3.1(0x0, 0x0, 0x0, 0x840130, 0xd, 0x0)
        /go/src/github.com/roboll/helmfile/main.go:188 +0xb8
main.findAndIterateOverDesiredStates(0x0, 0x0, 0xc420148af0, 0x0, 0x0, 0x0, 0x0, 0xc42004f350, 0x1, 0x1, ...)
        /go/src/github.com/roboll/helmfile/main.go:610 +0x414
main.findAndIterateOverDesiredStatesUsingFlags(0xc4200c8c80, 0xc420148af0, 0xc420148b10, 0x16)
        /go/src/github.com/roboll/helmfile/main.go:549 +0x278
main.main.func3(0xc4200c8c80, 0x0, 0x0)
        /go/src/github.com/roboll/helmfile/main.go:187 +0x46
reflect.Value.call(0x7b7240, 0x8593a0, 0x13, 0x839eef, 0x4, 0xc420148ee8, 0x1, 0x1, 0xc42000e100, 0x82d780, ...)
        /usr/local/go/src/reflect/value.go:447 +0x969
reflect.Value.Call(0x7b7240, 0x8593a0, 0x13, 0xc420148ee8, 0x1, 0x1, 0x4, 0xc420083438, 0x5)
        /usr/local/go/src/reflect/value.go:308 +0xa4
github.com/roboll/helmfile/vendor/github.com/urfave/cli.HandleAction(0x7b7240, 0x8593a0, 0xc4200c8c80, 0x0, 0x0)
        /go/src/github.com/roboll/helmfile/vendor/github.com/urfave/cli/app.go:483 +0x173
github.com/roboll/helmfile/vendor/github.com/urfave/cli.Command.Run(0x83a193, 0x4, 0x0, 0x0, 0x0, 0x0, 0x0, 0x851242, 0x35, 0x0, ...)
        /go/src/github.com/roboll/helmfile/vendor/github.com/urfave/cli/command.go:186 +0x999
github.com/roboll/helmfile/vendor/github.com/urfave/cli.(*App).Run(0xc420001200, 0xc42000e080, 0x4, 0x4, 0x0, 0x0)
        /go/src/github.com/roboll/helmfile/vendor/github.com/urfave/cli/app.go:237 +0x546
main.main()
        /go/src/github.com/roboll/helmfile/main.go:474 +0x1f79
```

--- a/main.go
+++ b/main.go
@@ -580,6 +580,8 @@ func findAndIterateOverDesiredStates(fileOrDir string, converge func(*state.Helm
 				switch stateLoadErr.Cause.(type) {
 				case *state.UndefinedEnvError:
 					noTarget = true
+				default:
+					return err
 				}
 			default:
 				return err

